jeison.el

```diff
@@ -26,14 +28,86 @@
 
 ;;; Commentary:
 
-;; Jeison is a library for transforming JSON objects (or `alist's) into
-;; EIEIO objects.
+;; 'jeison' is a library that transforms a JSON stream represented
+;; as an 'alist' or a naked 'alist' on its own into EIEIO/CLOS
+;; objects. New features since 1.0.0:
+;;
+;; - A 'vector-of' slot type constraint.  This is analogous to
+;;   'list-of'.
+;;
+;; - '(jeison-hash-table-of key-type element-type)' type constraint
+;;   maps JSON object member names to an 'element-type'. See the "Hash
+;;   Tables" documentation.
+;;
+;; - Uses the Emacs-native JSON parsing, if detected.  'jeison' will
+;;   fall back to the 'json.el' E-lisp package if native JSON support
+;;   is not available.
+;;
+;;   NOTE: 'jeison-false' is the meta-syntactic variable for JSON
+;;   'false' values -- native JSON defaults to ':false' whereas
+;;   'json.el' defaults to ':json-false'.  'jeison-false' is set to
+;;   the respective default, depending on the parser (either ':false'
+;;   or ':json-false'.) Use a let-binding around 'jeison-read' to
+;;   change 'jeison-false' to your preferred value when necessary.
+;;
+;; - A '(or null type)' type constraint is available, primarily to
+;;   accommodate optional values in the JSON input stream. You can use
+;;   ':initform nil' to initialize the slot; if the slot is not 'nil'
+;;   after reading the JSON stream, a value was present in the stream.
+;;
+;; - ':initform' is respected as the slot's default value if the input
+;;   JSON stream provides no replacement value. Alternatively, you can
+;;   use the '(or null type)' type constraint.
+
 
 ;;; Code:
+
 (require 'cl-lib)
 (require 'dash)
 (require 'eieio)
-(require 'json)
+
+(defvar jeison-debug-mode nil
+  "'jeison' debugging level.  0 = off, 5 = highest.")
```

Comment:
Seems to be unused

Reply:
It doesn't hurt to keep the `jeison-debug-mode` if it may be needed at a later time. Suggest keeping it.

Reply:
OK